### PR TITLE
[13.0][IMP] web_widget_one2many_product_picker: es translation

### PR DESCRIPTION
**Pull request by Tardo**
cc @tecnativa

### CODE REVIEW

**Review by sergio-teruel — APPROVED** [review]

**Comment by pedrobaeza** [comment]
This should be done on Weblate: https://translation.odoo-community.org/projects/web-13-0/web-13-0-web_widget_one2many_product_picker , and there shouldn't be any POT change.

File: web_widget_one2many_product_picker/i18n/es.po [hunk 1]
```diff
@@ -6,57 +6,65 @@ msgid ""
...
-"Content-Transfer-Encoding: \n"
+"Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=n != 1;\n"
-"X-Generator: Weblate 4.3.2\n"
+"X-Generator: Poedit 3.0\n"
 
 #. module: web_widget_one2many_product_picker
 #. openerp-web
-#: code:addons/web_widget_one2many_product_picker/static/src/xml/one2many_product_picker_quick_create.xml:6
+#: code:addons/web_widget_one2many_product_picker/static/src/xml/one2many_product_picker_quick_create.xml:0
```
Comment (pedrobaeza) [inline web_widget_one2many_product_picker/i18n/es.po:22]: This is very weird. Why this?

**Comment by pedrobaeza** [comment]
I'm reopening this as it seems POT is not being regenerated nor PO.

**Comment by pedrobaeza** [comment]
@Tardo can this be closed or is still applicable? Can you answer why the POT is different?

**Pull request closed without merging**